Makefile

```diff
@@ -299,14 +321,35 @@ build: fmt ## Build bpfman-operator and bpfman-agent binaries.
 # (i.e. docker build --platform linux/arm64 ). However, you must enable docker buildKit for it.
 # More info: https://docs.docker.com/develop/develop-images/build_enhancements/
 .PHONY: build-images
-build-images: ## Build bpfman, bpfman-agent, and bpfman-operator images.
-	docker build -t ${BPFMAN_OPERATOR_IMG} -f Containerfile.bpfman-operator ./
-	docker build -t ${BPFMAN_AGENT_IMG} -f Containerfile.bpfman-agent ./
+build-images: ## Build bpfman-agent, and bpfman-operator images.
+	trap 'exit' INT; \
+	$(foreach target,$(MULTIARCH_TARGETS),$(call build_target,${BPFMAN_OPERATOR_IMG},$(target),Containerfile.bpfman-operator)) \
+	$(foreach target,$(MULTIARCH_TARGETS),$(call build_target,${BPFMAN_AGENT_IMG},$(target),Containerfile.bpfman-agent))
 
 .PHONY: push-images
 push-images: ## Push bpfman-agent, bpfman-operator images.
-	docker push ${BPFMAN_OPERATOR_IMG}
-	docker push ${BPFMAN_AGENT_IMG}
+	trap 'exit' INT; \
+	$(foreach target,$(MULTIARCH_TARGETS),$(call push_target,${BPFMAN_OPERATOR_IMG},$(target))) \
+	$(foreach target,$(MULTIARCH_TARGETS),$(call push_target,${BPFMAN_AGENT_IMG},$(target)))
+
+.PHONY: manifest-build
+manifest-build: ## Build MULTIARCH_TARGETS manifest for bpfman-operator and bpfman-agent.
+	echo 'building manifest for $(BPFMAN_OPERATOR_IMG) and $(BPFMAN_AGENT_IMG)'
+	DOCKER_BUILDKIT=1 $(OCI_BIN) rmi ${BPFMAN_OPERATOR_IMG} -f
```

Comment:
What happens here if $(OCI_BIN) is podman? the `DOCKER_BUILDKIT=1` won't have any effect in that scenario right?

Reply:
Would it still work?

Reply:
OCI_BIN is docker if u have docker installed

Reply:
`DOCKER_BUILDKIT` is indeed ignored using podman however it works fine here (tested locally)

+ since docker engine 23, buildkit is the default behavior so we could even get rid of it if we mention this as prereqs in the README